src/frontend/src/game.js

```diff
@@ -0,0 +1,63 @@
+export class TokenKind {
```

Comment:
A bit surprising, I am assuming this comes from Unai's code for some reason? I don't know what this is doing here, but I will assume it's ok.

Reply:
I think it is Unai's code, I have no idea why it is here after I pull from dev and resolved the conflicts